src/Widget/BaseWidget.php

```diff
@@ -4,21 +4,29 @@
 
 namespace Bolt\Widget;
 
-use Bolt\Snippet\RequestZone;
-use Bolt\Snippet\Target;
 use Cocur\Slugify\Slugify;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Twig\Environment;
 
-class BaseWidget implements WidgetInterface
+/**
+ * BaseWidget can be used as easy starter pack or for as a base for heavy widgets.
```

Comment:
```suggestion
 * BaseWidget can be used as easy starter pack or as a base for widgets.
```

‘For’ is not needed, and `heavy` is highly subjective. ;-)